samples/interactive-tutorials/product/import-products-big-query-table.js

```diff
@@ -0,0 +1,94 @@
+// Copyright 2022 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main() {
+  // [START retail_import_products_from_big_query]
+
+  // Imports the Google Cloud client library.
+  const {ProductServiceClient} = require('@google-cloud/retail').v2;
+
+  const projectNumber = process.env['GCLOUD_PROJECT'];
+  const projectId = process.env['PROJECT_ID'];
+
+  const datasetId = 'products';
+  const tableId = 'products'; // TO CHECK ERROR HANDLING USE THE TABLE WITH INVALID PRODUCTS
+  const dataSchema = 'product';
+
+  // Placement
+  const parent = `projects/${projectNumber}/locations/global/catalogs/default_catalog/branches/default_branch`; // TO CHECK ERROR HANDLING PASTE THE INVALID CATALOG NAME HERE
+
+  // The desired input location of the data.
+  const inputConfig = {
+    bigQuerySource: {
+      projectId,
+      datasetId,
+      tableId,
+      dataSchema,
+    },
+  };
+
+  const reconciliationModes = {
+    RECONCILIATION_MODE_UNSPECIFIED: 0,
+    INCREMENTAL: 1,
+    FULL: 2,
+  };
+
+  const IResponseParams = {
+    IImportProductsResponse: 0,
+    IImportMetadata: 1,
+    IOperation: 2,
+  };
+
+  // The mode of reconciliation between existing products and the products to be imported.
+  const reconciliationMode = reconciliationModes.FULL;
```

Comment:
Should we start with INCREMENTAL? FULL may take longer to finish and may delete products created by other samples.

Reply:
done